run.sh

```diff
@@ -54,7 +54,14 @@ fi
 filename=$directory"/report_$(date +%s).json"
 
 nuclei -no-interactsh -disable-update-check -config $config -u $target -irr -json > $filename
-sed -i '' 's/}$/,"wafVersion":"'${wafVersion}'","nucleiVersion":"'${nucleiVersion}'","payloadVersion":"'${payloadVersion:="0"}'"}/g' $filename
+
+# check if we are using gnu sed
```

Comment:
Can you update the comment to read `check if using GNU sed, if not then -i requires passing an empty extension`.

Reply:
Updated